flag_engine/utils/json_encoder.py

```diff
@@ -0,0 +1,15 @@
+import json
+import decimal
+
+
+class DecimalEncoder(json.JSONEncoder):
+    """
+    Convert decimal to int/float because decimals are nothing but
+    int/float(for us) converted to decimal by boto3/dynamodb.
+    """
+    def default(self, obj):
+        if isinstance(obj, decimal.Decimal):
+            if obj.as_integer_ratio()[1] == 1:
```

Comment:
Is this the most efficient way to check if it's an integer? I'm not convinced. 

https://stackoverflow.com/questions/19965018/python-decimal-checking-if-integer